### PR TITLE
skeletal aws_vpcs

### PR DESCRIPTION
**Pull request by samcornwell**
Wrote skeletal aws_vpcs resource, tests, and doc.

#145

### CODE REVIEW

File: docs/resources/aws_vpcs.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+As this is the initial release of `aws_vpcs`, its limited functionality precludes examples.
+
+<br>
+
+## Matchers
```
Comment (clintoncwolfe) [inline docs/resources/aws_vpcs.md]: We generally add a sentence like "This resource defines no custom matchers."

File: libraries/aws_vpcs.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    end
+  '
+
+  # Underlying FilterTable implementation.
+  filter = FilterTable.create
+  filter.add_accessor(:entries)
+        .add(:exists?) { |x| !x.entries.empty? }
+  filter.connect(self, :access_key_data)
+
+  def access_key_data
```
Comment (clintoncwolfe) [inline libraries/aws_vpcs.rb]: This is probably copypasta from access_keys.  You should select a method name here that is specific to vpcs (`vpc_data`, perhaps)

**Review by clintoncwolfe — CHANGES_REQUESTED** [review]
Great contribution, @samcornwell !  Only two tiny changes requested.  

This is a great example of an easy-to-review, easy-to-approve skeletal resource.

File: docs/resources/aws_vpcs.md (hunk 1 above)
Comment (samcornwell) [reply docs/resources/aws_vpcs.md]: @clintoncwolfe I'm confused, I found 6 resources which only have an `exists` matcher, and none of them say that.

Comment (clintoncwolfe) [reply docs/resources/aws_vpcs.md]: I'm sorry, I misread this - I thought it was an empty Matchers section.  What you have here is just fine.

**Review by clintoncwolfe — APPROVED** [review]

**Pull request merged**